packages/shared/src/variants/parallel.ts

```diff
@@ -234,12 +229,10 @@ export class ParallelGo extends AbstractGame<
     });
   }
 
-  initializeParticipation(numPlayers: number): Participation[] {
-    const participation = new Array(numPlayers);
-    for (let i = 0; i < numPlayers; i++) {
-      participation[i] = { playerNr: i, dropOutAtRound: null };
-    }
-    return participation;
+  private participants() {
```

Comment:
Seems like we use `participants` more than `dropouts`. Might be a little bit simpler if we turn it around - save participants in a variable, and remove a player when they drop out.

Reply:
Good call! updated